.travis.yml

```diff
@@ -27,7 +27,7 @@ before_install:
   sudo mkdir -p /etc/containers/
   echo -e "[registries.search]\nregistries = ['docker.io']\n" | sudo tee /etc/containers/registries.conf
   echo '{ "features": { "buildkit": true } }' | sudo tee /etc/docker/daemon.json
```

Comment:
>> chrpath

Do I want to know?

Reply:
I don't think thats new, but it enables BuildKit in docker. The `sudo tee` is  farily standard way to write to files using `sudo` since something like: `sudo echo "foo" > bar.txt` doesn't work